documentation/4.0/content/managing-domains/model-in-image/usage.md

```diff
@@ -38,10 +38,16 @@ Model in Image requires an image with a WebLogic Server installation.
 Model in Image requires the following directory structure in its pods for
 its (optional) WDT model artifacts and (required) WDT binaries:
 
-| Domain resource attribute  | Default directory          | Contents                              |
-| -------------------------- | -------------------------- | ------------------------------------- |
-| `domain.spec.configuration.model.modelHome` | `/u01/wdt/models` | Zero or more model `.yaml`, `.properties`, and/or archive `.zip` files.|
-| `domain.spec.configuration.model.wdtInstallHome` | `/u01/wdt/weblogic-deploy` | Unzipped WDT installation binaries (required).  |
+| Domain resource attribute  | Contents                              | Default directory |
+| -------------------------- | ------------------------------------- | ----------------- |
+| `domain.spec.configuration.model.modelHome` | Zero or more model `.yaml`, `.properties`, and/or archive `.zip` files. | Optional. Location of the WDT model home, which can include model YAML files, `.properties` files, and application `.zip` archives. Defaults to `/u01/wdt/models` if no [Auxiliary Images]({{<relref "/managing-domains/model-in-image/auxiliary-images" >}}) are configured, and to `/aux/models` otherwise.|
+| `domain.spec.configuration.model.wdtInstallHome` | Unzipped WDT installation binaries (required).  | Optional. Location of the WDT installation. Defaults to `/u01/wdt/weblogic-deploy` if no [Auxiliary Images]({{<relref "/managing-domains/model-in-image/auxiliary-images" >}}) are configured, and to `/aux/weblogic-deploy` otherwise.|
+
+{{% notice note %}}
+If you set `modelHome` and `wdtInstallHome` to a non-default value,
+then the operator will ignore WDT model and installation files
```

Comment:
Should this say "domain" instead of the operator and "in Auxiliary images" similar to line 116-117 in `documentation/4.0/content/managing-domains/model-in-image/auxiliary-images.md` ?